lib/tasks/generate_test_data.rake

```diff
@@ -1,19 +1,6 @@
 desc 'Generate test applications for existing providers'
 task generate_test_applications: :environment do
-  GenerateTestApplications.new(for_year: :previous_year).perform
-
-  GetApplicationChoicesReadyToRejectByDefault.call.find_each do |choice|
```

Comment:
Removing this explicit rejection step between years means there will be previous year applications still awaiting provider decision. Could we add a rejection step somewhere?

Reply:
I've added it here: https://github.com/DFE-Digital/apply-for-teacher-training/pull/3190/files#diff-01a8ebf1b37797dc1ed6f0fcb98758a4ba7f9c9b91f1c5d449fa803ede34255cR37